test/e2e/framework.go

```diff
@@ -307,17 +307,40 @@ func (data *TestData) createVCNamespace(namespace string) error {
 	}()
 
 	svID, _ := data.vcClient.getSupervisorID()
+	storagePolicyID, _ := data.vcClient.getStoragePolicyID()
+	log.Info("", "storagePolicyID", storagePolicyID)
```

Comment:
It is strange to use an empty string in the log sentence?

Reply:
ok